src/routes/api/userRoute.js

```diff
@@ -0,0 +1,113 @@
+import express from 'express';
+import Validate from '../../middlewares/Validate';
+import isUserVerified from '../../middlewares/isUserVerified';
+import UserController from '../../controllers/UserController';
+import AuthenticateToken from '../../helpers/AuthenticateToken';
+
+
+const userRoute = express.Router();
+
+const { profileUpdateRules, validate } = Validate;
+
+/**
+ * @swagger
+ *
+ * /auth/google:
+ *    post:
+ *      summary: User login with google account
+ *      tags: [Users]
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/schemas/User'
+ *      responses:
+ *        "200":
+ *          Logged in with facebook successfully
+ */
+/**
+* @swagger
+*
+* /user/profile-settings:
+*   put:
+*     summary: User Profile Update
+*     description: Edit user profile information
+*     tags:
+*       - Profiles
+*     parameters:
+*      - name: token
+*        in: header
+*        required: true
+*        description: user token
+*        schema:
+*          type: string
+*        example: xxxxxxxxx.xxxxxxxxx.xxxxxxxxxx
+*        minimum: 1
+*     requestBody:
+*       content:
+*         application/json:
+*           schema:
+*             type: object
+*             properties:
+*               gender:
+*                 type: string
+*               birthDate:
+*                 type: string
+*               PreferedLanguage:
+*                 type: string
+*               PreferedCurrency:
+*                 type: string
+*               preferredLanguage:
+*                 type: string
+*               preferredCurrency:
+*                 type: string
+*               address:
+*                 type: string
+*               department:
+*                 type: string
+*               lineManager:
+*                 type: string
+*               bio:
+*                 type: string
```

Comment:
You are duplicating the schema. I recommend to create one schema and referencing it.